dev-lang/fpc/fpc-3.0.4.recipe

```diff
@@ -0,0 +1,145 @@
+SOURCE_DIR="fpcbuild-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	fpc$secondaryArchSuffix = $portVersion
+	cmd:fpc$secondaryArchSuffix = $portVersion
+	cmd:ppc386$secondaryArchSuffix = $portVersion
+	cmd:bin2obj$secondaryArchSuffix = $portVersion
+	cmd:chmcmd$secondaryArchSuffix = $portVersion
+	cmd:chmls$secondaryArchSuffix = $portVersion
+	cmd:data2inc$secondaryArchSuffix = $portVersion
+	cmd:delp$secondaryArchSuffix = $portVersion
+	cmd:fd2pascal$secondaryArchSuffix = $portVersion
+	cmd:fp$secondaryArchSuffix = $portVersion
+	cmd:fpcjres$secondaryArchSuffix = $portVersion
+	cmd:fpclasschart$secondaryArchSuffix = $portVersion
+	cmd:fpcmake$secondaryArchSuffix = $portVersion
+	cmd:fpcmkcfg$secondaryArchSuffix = $portVersion
+	cmd:fpcres$secondaryArchSuffix = $portVersion
+	cmd:fpcsubst$secondaryArchSuffix = $portVersion
+	cmd:fpdoc$secondaryArchSuffix = $portVersion
+	cmd:fppkg$secondaryArchSuffix = $portVersion
+	cmd:fprcp$secondaryArchSuffix = $portVersion
+	cmd:h2pas$secondaryArchSuffix = $portVersion
+	cmd:h2paspp$secondaryArchSuffix = $portVersion
+	cmd:instantfpc$secondaryArchSuffix = $portVersion
+	cmd:makeskel$secondaryArchSuffix = $portVersion
+	cmd:mkarmins$secondaryArchSuffix = $portVersion
+	cmd:mkinsadd$secondaryArchSuffix = $portVersion
+	cmd:mkx86ins$secondaryArchSuffix = $portVersion
+	cmd:pas2fpm$secondaryArchSuffix = $portVersion
+	cmd:pas2jni$secondaryArchSuffix = $portVersion
+	cmd:pas2ut$secondaryArchSuffix = $portVersion
+	cmd:plex$secondaryArchSuffix = $portVersion
+	cmd:postw32$secondaryArchSuffix = $portVersion
+	cmd:ppdep$secondaryArchSuffix = $portVersion
+	cmd:ppudump$secondaryArchSuffix = $portVersion
+	cmd:ppufiles$secondaryArchSuffix = $portVersion
+	cmd:ppumove$secondaryArchSuffix = $portVersion
+	cmd:ptop$secondaryArchSuffix = $portVersion
+	cmd:pyacc$secondaryArchSuffix = $portVersion
+	cmd:rmcvsdir$secondaryArchSuffix = $portVersion
+	cmd:rstconv$secondaryArchSuffix = $portVersion
+	cmd:unitdiff$secondaryArchSuffix = $portVersion
+	cmd:cvsco.tdf$secondaryArchSuffix = $portVersion
+	cmd:cvsdiff.tdf$secondaryArchSuffix = $portVersion
+	cmd:cvsup.tdf$secondaryArchSuffix = $portVersion
+	cmd:fp.ans$secondaryArchSuffix = $portVersion
+	cmd:gplprog.pt$secondaryArchSuffix = $portVersion
+	cmd:gplunit.pt$secondaryArchSuffix = $portVersion
+	cmd:grep.tdf$secondaryArchSuffix = $portVersion
+	cmd:makeskel.rsj$secondaryArchSuffix = $portVersion
+	cmd:program.pt$secondaryArchSuffix = $portVersion
+	cmd:ptop.rsj$secondaryArchSuffix = $portVersion
+	cmd:tpgrep.tdf$secondaryArchSuffix = $portVersion
+	cmd:unit.pt$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix  >= 5.9
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel >= r1~alpha4_pm_hrev51411
+	"
+BUILD_PREREQUIRES="
+	binutils$secondaryArchSuffix
+	cmd:tar
+	cmd:make
+	cmd:diff
+	cmd:find
+	fpc_bin == 3.0.2
```

Comment:
I think this should be `cmd:fpc >= 3.0.2` instead.

Reply:
No, it should really be 3.0.2. Freepascal is only guaranteed to compile with the previous official version at the time of release (see the top of https://freepascal.org/develop.var).